queries.sql

```diff
@@ -5,4 +5,57 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered='True';
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Day 2 */
+
+BEGIN;
+
+UPDATE animals SET species='unspecified';
+
+SELECT species FROM animals;
+ROLLBACK;
+
+SELECT species FROM animals;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon';
+
+COMMIT;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```

Comment:
- [ ] You have created a query great but it would be nice if you would verify your changes before and after Rollback 👍
Your query will look something like this ⬇️
```sql
   BEGIN;

   DELETE FROM animals;
   SELECT * FROM ANIMALS;

   ROLLBACK;
   SELECT * FROM ANIMALS;
```

Reply:
Modified everything following the review